camayoc/constants.py

```diff
@@ -291,7 +291,7 @@
 MASKED_PASSWORD_OUTPUT = '\*{8}'
 """Regex that matches password on outputs."""
 
-CONNECTION_PASSWORD_INPUT = 'Provide connection password.\r\nPassword:'
+CONNECTION_PASSWORD_INPUT = 'Provide( a)? connection password.\r\nPassword:'
```

Comment:
` a` is not displayed in rho but it is in quipucords. This will ensure the same constant can be used for both.